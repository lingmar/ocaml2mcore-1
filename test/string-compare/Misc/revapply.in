let add x y = x + y in
2 |> add 1
