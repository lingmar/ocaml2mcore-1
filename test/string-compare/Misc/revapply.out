mexpr
let add = lam x. lam y.
  addi x y
in add 1 2
